docs/components/datetimepicker.md

```diff
@@ -0,0 +1,407 @@
+# DateTime Picker
+
+<div v-if="hasMounted">
+
+**KDateTimePicker** - A date and time selection tool, displayed inside a popover.
+
+## Examples
+
+A single date or time picker instance will emit a Jasvscript Date object as a `string`.
+Whereas an instance of this component that displays a date range or a series of relative time frames, will emit an `object` containing the following values:
+
+```js
+{ 
+  "start": "2022-09-06T19:45:54.406Z",
+  "end": "2022-09-07T19:45:54.406Z",
+  "timePeriodsKey": "24h" // or ""
+}
+```
+
+### Single date
+
+Create a single date selection calendar, defaulting the [selected value](#defaultcustom) to tomorrow.
```

Comment:
I think it should default to the current day, not tomorrow. This matches other popular UI libraries.